package.json

```diff
@@ -18,6 +18,8 @@
     "moment": "^2.29.4",
     "react": "^18.2.0",
     "react-bootstrap": "^2.9.1",
+    "react-dnd": "^16.0.1",
```

Comment:
if one of these packages are not used anymore, you can remove it by running the command:
`npm uninstall "packageName"`